lib/slack-notifier/util/http_client.rb

```diff
@@ -21,7 +23,14 @@ def initialize uri, params
         end
 
         def call
-          http_obj.request request_obj
+          response = http_obj.request(request_obj)
```

Comment:
I personally prefer not to use `tap`, because it feels like control flow with the block and everything. But if we still want to return the response so that currently working code won't break I think I'm okay to change it! :grin: